kinetic/distribution.yaml

```diff
@@ -10314,6 +10314,22 @@ repositories:
       url: https://github.com/ros/resource_retriever.git
       version: kinetic-devel
     status: maintained
+  rexrov2:
+    release:
+      packages:
+      - rexrov2_control
+      - rexrov2_description
+      - rexrov2_gazebo
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/uuvsimulator/rexrov2-release.git
+      version: 0.1.3-0
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/uuvsimulator/rexrov2.git
+      version: 0.1.3
```

Comment:
Looks like CI failed because the `source` -> `version` must be a branch, but `0.1.3` is a tag. It looks like this should be `master` instead. There's an explanation in this pull request: https://github.com/ros/rosdistro/pull/20286/files

Reply:
```suggestion
      version: master
```